brix/ciaox11/apc/project.rb

```diff
@@ -350,6 +350,31 @@ def add_recipe_file (path, implicit=false)
       recipe_files[path] = frcp
     end
 
+    # retrieve enabled features
+    def features
+      unless @features
+        @features = {}
+        _ace_root = ::BRIX11::Exec.get_run_environment('ACE_ROOT')
+        _cfg_file = File.join(_ace_root, 'bin', 'MakeProjectCreator', 'config', 'default.features')
+        if File.exist?(_cfg_file)
+          File.readlines(_cfg_file).each do |ln|
+            if /\A\s*\w+\s*=\s*1\s*\Z/ =~ ln  # only match enabled features
+              feature = ln.split('=').first.strip
+              @features[feature.to_sym] = true
+            end
```

Comment:
For a feature, the last setting will win in MPC, so at the moment there is a =0 we should remove the feature when it is already there, for example this could happen

```
dds4ccm=1
dds4cc=0
```

Reply:
BRIX11 generates feature files without this kind of errors. If user hacks them and makes mistakes it's a user error.